build/azure-pipelines/product-compile.yml

```diff
@@ -70,6 +70,13 @@ steps:
     displayName: Install dependencies
     condition: and(succeeded(), ne(variables.NODE_MODULES_RESTORED, 'true'))
 
+  # Ref https://github.com/microsoft/vscode/issues/189019
+  - ${{ if eq(parameters.VSCODE_QUALITY, 'oss') }}:
```

Comment:
The change should be after https://github.com/microsoft/vscode/blob/2bba361d30764a1f02a0b1e7513e5c7bd5853758/build/azure-pipelines/linux/product-build-linux.yml#L153 , so that it gets executed in linux test pipeline.